test/SILGen/dynamically_replaceable.swift

```diff
@@ -9,22 +9,22 @@ defer {
 }
 
 // CHECK-LABEL: sil hidden [ossa] @$s23dynamically_replaceable014maybe_dynamic_B0yyF : $@convention(thin) () -> () {
-// IMPLICIT-LABEL: sil hidden [dynamically_replacable] [ossa] @$s23dynamically_replaceable014maybe_dynamic_B0yyF : $@convention(thin) () -> () {
+// IMPLICIT-LABEL: sil hidden [dynamically_replaceable] [ossa] @$s23dynamically_replaceable014maybe_dynamic_B0yyF : $@convention(thin) () -> () {
```

Comment:
@AnthonyLatsis I guess I should revert ALL changes to `dynamically_replaceable` in this PR as well?

Reply:
Yeah, I suggest renaming the attribute in a separate patch.

Reply:
reverted